tests/foreman/cli/test_repository.py

```diff
@@ -361,8 +523,15 @@ def test_positive_update_url(self):
         @Assert: Repository url is updated
         """
         new_repo = self._make_repository()
+        # generate a tuple of auth repos with all valid credentials
+        auth_repos = ()
```

Comment:
Please note, that the only "added value" of this feature is to accept repo URL in more formats than before. In scope of updating the URL there is no difference between `http://foo.bar/baz` and `http://monty:python@foo.bar/baz` - the update feature takes both as evenly valid input values so I don't see why the latter one should deserve a special unit test.